contracts/Governance/RadicleToken.sol

```diff
@@ -187,6 +187,35 @@ contract RadicleToken {
         return true;
     }
 
+    function burnFrom(address account, uint256 rawAmount) public virtual {
+        require(account != address(0), "RadicleToken::burnFrom: cannot burn from the zero address");
+        uint96 amount = safe96(rawAmount, "RadicleToken::approve: amount exceeds 96 bits");
```

Comment:
```suggestion
        uint96 amount = safe96(rawAmount, "RadicleToken::burnFrom: amount exceeds 96 bits");
```

Reply:
Just noticed this one

Reply:
Fixed. I've double checked all the new error messages, that was the last issue.